src/muse/readers/csv.py

```diff
@@ -403,6 +400,10 @@ def read_technologies(
     if "comm_type" in result.data_vars or "comm_type" in result.coords:
         result = result.drop_vars("comm_type")
 
+    check_utilization_and_minimum_service_factors(
+        result.to_dataframe(), [tpath, ttpath]
```

Comment:
Ideally we should only include `ttpath` is it's not `None`, otherwise we get `None` printed as a path in the error message

Reply:
sorted